deploy/provider/docker-for-mac/service.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+  labels:
+    app: ingress-nginx
+spec:
+  externalTrafficPolicy: Local
```

Comment:
Probably not necessary here. Docker for Mac is a single-instance cluster.

Reply:
Good catch! Fixed.